examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,19 @@
+# Generated folders 
+build/
+node_modules/
```

Comment:
Well, if you're planning to manage front-end dependencies using npm, which you should if it's possible (#1110), ignoring it isn't going to work :)